projectgen-micronaut-openrewrite/src/main/resources/META-INF/rewrite/microstream-rest.yml

```diff
@@ -0,0 +1,35 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: io.micronaut.starter.feature.microstream-rest
+displayName: Add Microstream Rest dependency and link to README
+recipeList:
+  - io.micronaut.starter.feature.microstream-rest.dependencies
+  - io.micronaut.starter.feature.microstream-rest.micronaut.documentation
+  - io.micronaut.starter.feature.microstream-rest.documentation.thirdparty
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.micronaut.starter.feature.microstream-rest.dependencies
+displayName: Add Microstream Rest dependency
+recipeList:
+  - org.openrewrite.java.dependencies.AddDependency:
+      groupId: io.micronaut.microstream
+      artifactId: micronaut-microstream-rest
+      configuration: devlopmentOnly
+      scope: compile
```

Comment:
In maven this is `provided`. Please, be careful with the details: 
https://micronaut.io/launch?type=DEFAULT&name=demo&package=com.example&javaVersion=JDK_21&lang=JAVA&build=MAVEN&test=JUNIT&features=microstream-rest&version=4.8.2
```
    <dependency>
      <groupId>io.micronaut.microstream</groupId>
      <artifactId>micronaut-microstream-rest</artifactId>
      <scope>provided</scope>
    </dependency>
```